openedx/core/djangoapps/verified_track_content/admin.py

```diff
@@ -4,8 +4,8 @@
 
 from django.contrib import admin
 
-from verified_track_content.forms import VerifiedTrackCourseForm
-from verified_track_content.models import VerifiedTrackCohortedCourse
+from openedx.core.djangoapps.verified_track_content.forms import VerifiedTrackCourseForm
```

Comment:
Move of verified_track_content from LMS to openedx because of CMS unit test dependencies.